ui/src/tool-panel/switch/switch-joint-infobox.tsx

```diff
@@ -32,10 +36,27 @@ const SwitchJointInfobox: React.FC<SwitchJointInfobox> = ({
         ),
     );
 
-    const locationTracks = useLoader(
-        () => getLocationTracksForJointConnections(publishType, jointConnections),
-        [switchAlignments, jointConnections, publishType],
-    );
+    const displayedLocationTracksEndingAtJoint =
+        topologicalJointConnections ?? locationTracksEndingAtJoint;
```

Comment:
Tietonäkymässä näytetään dataa locationTracksEndingAtJointin avulla, linkitysnäkymässä topologicalJointConnectionsin avulla. Näitä lienee mahdollista yhtenäistää edelleen, mutta jätin toistaiseksi tähän muotoon.